src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/matching_indices_list/matching_indices_list.html

```diff
@@ -53,7 +53,7 @@
         class="kuiMenuItem"
         ng-repeat="index in matchingIndicesList.pageOfIndices"
       >
-        <p class="kuiText" ng-bind-html="index.name | highlight:matchingIndicesList.formattedPattern">
+        <p class="kuiText" ng-click="matchingIndicesList.clickPattern(index.name)" ng-bind-html="index.name | highlight:matchingIndicesList.formattedPattern">
```

Comment:
Can you take a look at the accessibility style guide and apply its guidance here? https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#use-button-and-a-href